### PR TITLE
Fix multi master Helm conditions

### PR DESCRIPTION
**Pull request by ar3ndt**

### CODE REVIEW

**Comment by toszo** [comment]
Please link issues with this PR

File: core/src/epicli/data/common/ansible/playbooks/roles/helm/tasks/main.yml [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
+    tasks_from: gather-facts
+
 - name: Check if Helm chart repo already exists
   shell: helm repo list | grep {{ helm_chart_repo_name }}
   args:
     executable: /bin/bash
   register: helm_repo_list
   failed_when: (helm_repo_list.rc != 0 and not 'no repositories' in helm_repo_list.stderr) 
                or helm_repo_list.rc > 1
+  when: inventory_hostname == target_repository_hostnames[0]
```
Comment (toszo) [inline core/src/epicli/data/common/ansible/playbooks/roles/helm/tasks/main.yml:14]: Playing on all hosts but some tasks should be executed only on some of them, it is not clear I think, please comment.

**Comment by sk4zuzu** [comment]
@toszo please take a look at this PR #1360, it has some more fixes and removes execution on all hosts.

**Comment by mkyc** [comment]
Does this PR solve some issue? Should it be linked?

**Comment by mkyc** [comment]
@ar3ndt should this PR be abandoned due to #1360 or does it provide something not covered there?

File: core/src/epicli/data/common/ansible/playbooks/roles/helm/tasks/main.yml (hunk 1 above)
Comment (ar3ndt) [reply core/src/epicli/data/common/ansible/playbooks/roles/helm/tasks/main.yml:14]: this PR is already covered by other one from sk4zuzu

**Comment by ar3ndt** [comment]
> @ar3ndt should this PR be abandoned due to #1360 or does it provide something not covered there?

yes

**Pull request closed without merging**